src/js/components/put-away/PutAwayCheckPage.jsx

```diff
@@ -177,6 +182,21 @@ class PutAwayCheckPage extends Component {
    * @public
    */
   completePutAway() {
+    let isBinLocationChosen = true;
+    _.some(this.props.putAway.putawayItems, (putAwayItem) => {
```

Comment:
@DominikaHatala _.some is returning a value
`const isBinLocationChosen = !_.some(this.props.putAway.putawayItems, putAwayItem => _.isNull(putAwayItem.putawayLocation.id)`